.sqlfluff

```diff
@@ -0,0 +1,47 @@
+[sqlfluff]
+dialect = postgres
+fix_even_unparsable = True
```

Comment:
Unfortunately, ALL of our files that contain `{{schema | sqlsafe}}` are unparsable because SQLFluff doesn't seem to know what to do with the `sqlsafe` bit. 

example output:
```
L: 572 | P:  17 |  PRS | Line 572, Position 17: Found unparsable section:
                       | '.{{schema | sqlsafe}}.reporting_ocpawsco...'
```

https://github.com/sqlfluff/sqlfluff/issues/3074

Reply:
Oh wow. That's unfortunate. Leave a comment about why we have to enable this since it is highly discouraged in the docs.